docs/standards/generic-standards/lsp14-ownable-2-step.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: 'LSP14 - Ownable 2-Step'
+sidebar_position: 3
+---
+
+# LSP14 - Ownable 2-Step
+
+:::info Standard Document
+
+[LSP14 - Ownable 2-Step](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-14-Ownable2Step.md)
+
+:::
+
+## Introduction
+
+The **[LSP14 Standard](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-14-Ownable2Step.md)** describes an extended version of **[EIP173](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-173.md)** that uses a 2-step process to transfer or renounce ownership of a contract, instead of instant execution.
```

Comment:
This is the same content as what is in the Introduction section of the LSP14 standards specs.

I don't think we should have exactly the same content as the standards specs, but rather a short and layman version. A good headline would be:

- _what is the existing problem of the Ownable standard?_
- _what is the proposed solution using LSP14?_

And in the section **## What does this standard represent ?** I would describe:

_How LSP14 solves it._